src/pages/Research/Content/ResearchListItem.tsx

```diff
@@ -198,7 +198,8 @@ const getItemDate = (item: IResearch.ItemDB, variant: string): string => {
 const calculateTotalComments = (item: IResearch.ItemDB) => {
   if (item.updates) {
     const commentOnUpdates = item.updates.reduce((totalComments, update) => {
-      const updateCommentsLength = update.comments ? update.comments.length : 0
+      const updateCommentsLength =
+        !update._deleted && update.comments ? update.comments.length : 0
```

Comment:
```suggestion
        !update._deleted && update.status !== 'draft' && update.comments ? update.comments.length : 0
```

**Issue**: we are also not checking if the update status is NOT a draft, can we please add this check or else we will be showing the wrong number of comments.